extensions/amp-access/0.1/access-expr-impl.jison

```diff
@@ -0,0 +1,130 @@
+
```

Comment:
If you don't mind, I'd rather do it in a separate PR later. For now, I rather appreciate being able to do diffs between them.